config/config.go

```diff
@@ -0,0 +1,350 @@
+package config
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"os"
+	"sort"
+	"strings"
+
+	"github.com/taskcluster/taskcluster-cli/extpoints"
+	"github.com/taskcluster/taskcluster-client-go/tcclient"
```

Comment:
This should be `tcclient "github.com/taskcluster/taskcluster-client-go"`. You probably need to run `go get -u` as it looks like you've developed against an older version of the client.